doc/source/examples/rpt/photon-count-calculation-in-a-cylindrical-vessel/photon-count-calculation-in-a-cylindrical-vessel.rst

```diff
@@ -0,0 +1,325 @@
+	
+
+where
+
+- :math:`\Omega` is the closed exposed surface of the detector;
+- :math:`\vec{r}` is a vector going from the position of the tracer particle (:math:`\vec{X}`) to a variable point (:math:`\vec{P}`) on the exposed surface of the detector;
+- :math:`d\vec{A}` is the external surface vector normal to the surface at the contact point on the detector's crystal;
+- :math:`f_a(\alpha, \theta)` is the probability function of the non-interaction between the :math:`\gamma`-rays emitted within :math:`\Omega` and the material inside the vessel, and
+- :math:`f_d(\alpha, \theta)` is the probability function of the interaction of the :math:`\gamma`-rays with the detector. 
+
+The two last functions may be re-written the following way :
+
+.. math::
+
+    f_a(\alpha, \theta) = exp\{-\mu_r \ e(\alpha, \theta)\}
+
+where :math:`\mu_r` is the reactor's attenuation coefficient and :math:`e(\alpha, \theta)` is the length of the path traveled by the photon inside the vessel.
+
+
+And
+
+.. math::
+
+    f_d(\alpha, \theta) = 1 - exp\{ -\mu_d \ d(\alpha,\theta)\}
+
+where :math:`\mu_d` is the detector's attenuation coefficient and :math:`d(\alpha,\theta)` is the length of the path traveled by the photon inside the detector.
+
+
+:raw-html:`<br />`
+
+
+Using the Monte Carlo algorithm, we approximate the previous closed surface integral by randomly selecting several thousands of photon path directions.
+
+Thus, the efficiency of the :math:`i_{th}` detector is calculated as follows :
+
+.. math::
+
+    \xi_i (\vec{X}) = \frac{1}{N} \sum_{j=1}^{N} \omega(\alpha) \omega(\theta) f_a(\alpha_j, \theta_j) f_d(\alpha_j, \theta_j)
+
+
+where
+
+- :math:`N` is the number of randomly generated photons;
+- :math:`\alpha_j` and :math:`\theta_j` are randomly generated angles that describe the direction of a ray emitted by a tracer particle;
+- :math:`\omega(\alpha)` is the weighting factor associated with the angle :math:`\alpha`, and
+- :math:`\omega(\theta)` is the weighting factor associated with the angle :math:`\theta`.
+
+
+:raw-html:`<br />`
+
+Parameter file
+----------------
+
+RPT Parameters
+~~~~~~~~~~~~~~~
+
+
+In the subsection *"rpt parameters"*, we define the values of the set of parameters necessary for calculating the counts using the Monte Carlo method.  Among these parameters, we have the name of the file in which is found a set of different positions of the particle inside the vessel (``particle position file``), the number of Monte Carlo iterations (``monte carlo iteration``), the seed that is used to generate a random number (``random number seed``) and other parameters that describe the studied :math:`\gamma`-ray model. We also define the name of the file in which the counts for each position will be exported in with the parameter ``counts file``. These common parameters used for the RPT simulation are described in the `RPT parameters <../../../parameters/rpt/rpt_parameters.html>`_ documentation page.
```

Comment:
Among these parameters, we have the name of the file in which contains a set of different positions of the particle inside the vessel